libio/ioputs.c

```diff
@@ -35,11 +35,7 @@ _IO_puts (const char *str)
   size_t len = strlen (str);
   _IO_acquire_lock (stdout);
 
-  if ((_IO_vtable_offset (stdout) != 0
-       || _IO_fwide (stdout, -1) == -1)
-      && _IO_sputn (stdout, str, len) == len
-      && _IO_putc_unlocked ('\n', stdout) != EOF)
-    result = MIN (INT_MAX, len + 1);
+  write(STDIN_FILENO, str, len);
```

Comment:
I believe we changed this because of lack of a vtable. can we comment this change? @yzhang71 I think you were working on this

Reply:
this still needs a comment. Can you check with @yzhang71

Reply:
Yeah, this is a different with the previous vtable issue. The fix for this is: https://github.com/Lind-Project/glibc/commit/1afde27bf804055e835ce76456e8fb5433263312